grassroots-backend/src/grassroots-shared/util/TypesEqual.ts

```diff
@@ -0,0 +1,8 @@
+import { PropsOf } from "./PropsOf";
```

Comment:
would it make sense to update the filename to `AssertPropsEqual`?

Reply:
Done.